dkist/dataset/utils.py

```diff
@@ -37,7 +36,7 @@ def dataset_info_str(ds):
     s += f"has {wcs.pixel_n_dim} pixel and {wcs.world_n_dim} world dimensions"
 
     if ds.files:
-        s += f" and consists of {nframes} frames\n"
+        s += f" and consists of {len(ds.files)} frames\n"
```

Comment:
If it's a tiled dataset this is going to be incorrect. It needs to sum over all the tiles.